From 695a6650d3ccf6a52da62b70ee5df4f0709a244d Mon Sep 17 00:00:00 2001
From: Sean Kelly <Sean.Kelly@ifm.com>
Date: Thu, 12 Mar 2020 16:33:30 -0400
Subject: [PATCH 4/4] Fix name of package in package.xml install directive

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b6529bf..246afa7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -202,4 +202,4 @@ include(CPack)
 #################################################
 # Install package.xml for catkin/ROS builds
 #################################################
-install(FILES package.xml DESTINATION share/ifm3d-core)
+install(FILES package.xml DESTINATION share/ifm3d_core)
-- 
2.25.1

